pkg/test/echo/docker/Dockerfile.app_sidecar

```diff
@@ -7,15 +7,8 @@ COPY certs/                           /var/lib/istio/
 COPY certs/default/*                  /etc/certs/
 
 # Install the sidecar components
-COPY envoy                            /usr/local/bin/envoy
-COPY pilot-agent                      /usr/local/bin/pilot-agent
-COPY node_agent                       /usr/local/bin/node_agent
-COPY istio-start.sh                   /usr/local/bin/istio-start.sh
-COPY istio-node-agent-start.sh        /usr/local/bin/istio-node-agent-start.sh
-COPY istio-iptables                   /usr/local/bin/istio-iptables
-COPY istio-clean-iptables             /usr/local/bin/istio-clean-iptables
-COPY postinst.sh                      /usr/local/bin/postinst.sh
-COPY envoy_bootstrap_v2.json          /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
+COPY istio-sidecar.deb  /tmp/istio-sidecar.deb
+RUN dpkg -i /tmp/istio-sidecar.deb && rm /tmp/istio-sidecar.deb
```

Comment:
Proxy already publishes several builds as docker containers: https://github.com/istio/proxy/blob/master/scripts/release-binary.sh#L152. Maybe we should use that instead of debian package? Using pre-build images is going to speed up the build.

Reply:
We don't properly document debian dependencies so I'm a bit concerned of libc ABI mismatch, for example. I think we should leave base OS selection to the proxy.

Reply:
istio-sidecar.deb is a package for VMs with iptables, nodeagent, pilotagent, envoy, etc. So I don't think we can just use the one from proxy

Reply:
And the point of this change is to test what VM users are using

Reply:
Note there is another Dockerfile under tools/packaging/.../deb that does the same thing.

Maybe we should remove one ?

Reply:
I'm pretty sure we'll want Asan/Tsan enabled deb packages down the line, so you would need to do the same thing as the proxy does with all the various builds.